web/src/components/layout/layout.tsx

```diff
@@ -31,7 +33,13 @@ import {
   challengeTeamTokens,
   ChallengeToken,
   challengeTokens,
+  FeatureType,
+  FeatureToken,
+  features,
 } from 'common';
+import API from '../../services/api';
+import NotificationBanner from './../notification-banner/notification-banner';
+import { Notifications } from '../../stores/notifications';
```

Comment:
I'm not using the Notification.add actions, because those notifications are not really meant to be persistent, and they are identified by ID, so every time you clicked into a page that used layout.tsx, went to speak.tsx, and came back there would be another banner, and no way to check for uniqueness (as far as I could tell) because the banner IDs just auto-increment. This solution is kind of wonky, but I think is the lowest overhead solution. I think there might be value in separating banners entirely from notifications, but I wasn't sure if at some point in the future we may want temporary banners?

Reply:
I think separating banners from notifications makes sense. This is a rough patch in our code already, so would you mind leaving a TODO (or Jira ticket or something) to capture that there's work to be done here?